proof/invariant-abstract/AInvsToplevel_AI.thy

```diff
@@ -0,0 +1,17 @@
+(*
+ * Copyright 2025, Proofcraft Pty Ltd
+ *
+ * SPDX-License-Identifier: GPL-2.0-only
+ *)
+
+theory AInvsToplevel_AI
+imports
+  ArchAInvsToplevel_AI
+  KernelInit_AI
+begin
+
+(* Remove this rule from [wp] here because it is not safe for proofs that involve extended state. It
+   is also removed in Deterministic_AI but may have been added back in during a theory merge. *)
+declare dxo_wp_weak[wp del]
```

Comment:
Still feels weird that it can't be used only in the theories that need it, but leaks out like this instead.